cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraClusterInfo.java

```diff
@@ -43,20 +43,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import o.a.c.sidecar.client.shaded.common.NodeSettings;
```

Comment:
the new version of Sidecar relocates the common dependencies to o.a.c.sidecar.client.shaded . See https://issues.apache.org/jira/browse/CASSANDRASC-104